wdio.conf.js

```diff
@@ -11,7 +11,7 @@ const webpackPort = 8080;
 // Flex specs search between local pacakge and repo
 let specs = path.join('tests', 'wdio', '**', '*-spec.js');
 if (__dirname === process.cwd()) {
-  specs = path.join('packages', '**', specs);
+  specs = path.join('packages', '*', specs);
```

Comment:
so this fixed a nasty issue I discovered where a double glob was causing wdio to pull in tests from the node_modules directory of every package. This means that when I pushed base tests, every component that used base reran those tests. Switching to a single glob excludes those node_module directories.

Reply:
@emilyrohrbough if any other projects integrated webdriver.io they may want to update this

Reply:
@mhemesath I'll work on getting the clinical PR updated